include/mbedtls/ctr_drbg.h

```diff
@@ -110,7 +106,9 @@ typedef struct
     unsigned char counter[16];  /*!< The counter (V). */
     int reseed_counter;         /*!< The reseed counter. */
     int prediction_resistance;  /*!< This determines whether prediction
-                                     resistance is enabled. */
+                                     resistance is enabled, that is
+                                     to systematically reseed before
```

Comment:
Suggestion: The two half sentences don't seem to fit together for me. Suggestion, based on @gilles-peskine-arm's wording: `This determines whether to enable prediction resistance, that is, whether to systematically reseed before each random generation.`

Reply:
Good idea.